File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        
        String[] arr = s.split(" ");
        
        
        for (String str : arr){
            if (str.length() == 0) answer += " ";
            else {
                answer += str.substring(0, 1).toUpperCase();
                answer += str.substring(1, str.length()).toLowerCase();
                answer += " ";
            }
        }
        
        if (s.substring(s.length() - 1, s.length()).equals(" ")) return answer;
        else return answer.substring(0, answer.length()-1);
    }
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    answer = ''
    
    a = s.split(" ")
    
    for k in a:
        if k:
            answer += (k[0].upper() + k[1:].lower())
            answer += " "
        else:
            answer += " "
    
    
    
    return answer[:len(answer)-1]